Foreground.skin.php

```diff
@@ -199,11 +199,11 @@ public function execute() {
 		<div class="row">
 				<div id="p-cactions" class="large-12 columns">
 					<?php if ($wgUser->isLoggedIn() || $wgForegroundFeatures['showActionsForAnon']): ?>
-						<a href="#" data-dropdown="drop1" class="button dropdown small secondary radius"><i class="fa fa-cog"><span class="show-for-medium-up">&nbsp;<?php echo wfMessage( 'actions' )->text() ?></span></i></a>
+						<a href="#" class="button dropdown small secondary radius"><i class="fa fa-cog"><span class="show-for-medium-up">&nbsp;<?php echo wfMessage( 'actions' )->text() ?></span></i></a>
```

Comment:
Missing the `data-options=` which is what caused issues recently with 1.2. This is what I use on a production wiki running 1.2 with Foundation 4. This should carry forward to 5 without any problems and you can keep the classes you tweaked here. Those are just style.

```
<a href="#" data-dropdown="hover1" data-options="is_hover:true; ignore_repositioning;" class="button dropdown small secondary radius"><i class="fa fa-cog"><span class="show-for-medium-up">&nbsp;<?php echo wfMessage( 'actions' )->text() ?></span></i></a>
	<!--RTL -->
	<ul id="hover1" data-dropdown-content="content" aria-hidden="true" class="views large-12 columns right f-dropdown">
		<?php foreach( $this->data['content_actions'] as $key => $item ) { echo preg_replace(array('/\sprimary="1"/','/\scontext="[a-z]+"/','/\srel="archives"/'),'',$this->makeListItem($key, $item)); } ?>
		<?php wfRunHooks( 'SkinTemplateToolboxEnd', array( &$this, true ) );  ?>
	</ul>
```